geoapps/utils/utils.py

```diff
@@ -41,6 +41,37 @@
 from sklearn.neighbors import KernelDensity
 
 
+def sorted_children_dict(object: UUID | Entity):
+
+    if isinstance(object, UUID):
+        object = workspace.get_entity(object)[0]
+
+    children_dict = {}
+    iters = []
+    iteration_data = []
+    other_data = []
+    for c in object.children:
+        if not isinstance(c, (IntegerData, FloatData)):
+            continue
+        else:
+            name = c.name
+            children_dict[name] = c.uid
+            iter = re.findall(r"[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?", name)
+            if iter:
+                try:
+                    iters.append(int(iter[0]))
+                except:
```

Comment:
Surprised that mypy didn't complain about using bare except. Is there a reason we would need to operate on integer only? I would suggest always using float and removing the try/except if not necessary.

Reply:
removed